sound/soc/sof/loader.c

```diff
@@ -20,13 +20,21 @@ static int get_ext_windows(struct snd_sof_dev *sdev,
 {
 	const struct sof_ipc_window *w =
 		container_of(ext_hdr, struct sof_ipc_window, ext_hdr);
+	size_t w_size = struct_size(w, window, w->num_windows);
+
+	if (sdev->info_window) {
+		if (memcmp(sdev->info_window, w, w_size)) {
+			dev_err(sdev->dev, "error: receive diverged window descriptions");
```

Comment:
What does diverged window desc mean? Does it mean there's a mistmatch between what was sent and what was received?

Reply:
changed